web/src/routes/OrganizationsList/Organization/Tabs/DefaultPermissions/DefaultPermissionsToolbar.tsx

```diff
@@ -52,6 +56,18 @@ export default function DefaultPermissionsToolbar(
           >
             Create default permission
           </Button>
+          <Conditional if={props.selectedItems?.length !== 0}>
+            <ToolbarItem>
+              <Button
+                style={{paddingBottom: '0px'}}
+                icon={<TrashIcon style={{fontSize: '24px'}} />}
+                isLarge={true}
```

Comment:
Getting an error saying this attribute doesn't exist. Also you can set boolean attributes to true just be adding them. 
`<Button isLarge ... />`